nautobot/extras/api/serializers.py

```diff
@@ -995,7 +1061,7 @@ class Meta:
         ]
 
 
-class RelationshipAssociationSerializer(ValidatedModelSerializer):
+class RelationshipAssociationSerializer(ValidatedModelSerializer, NotesSerializerMixin):
```

Comment:
`nautobot.extras.models.relationships.RelationshipAssociation` don't support notes. Do we need to add the notes mixin to the model or remove it from the serializer?

Reply:
If RelationshipAssociation should have notes, then we can add the Mixin to the Class.  I don't believe ReleationshipAssociations would require notes though, the model is not displayed to the end user.

Reply:
Yeah, I don't think notes make sense for RelationshipAssociation.

Reply:
It looks like this one still needs to be addressed

Reply:
Thanks, I missed resolving this one. Pushed a new commit to resolve this.